modules/assim.batch/R/gelman_diag.R

```diff
@@ -70,7 +70,8 @@ gelman_diag_mw <- function(x,
 #' more conservative approach than the moving-window method.
 #' @export
 gelman_diag_gelmanPlot <- function(x, ...) {
-  png("/dev/null")
+  #png("/dev/null")
```

Comment:
I would just delete this line to avoid confusing people.

Reply:
Working on my mac now, I need to test on a Linux machine next.  Also, I need to remove the commented part